pkg/controller/status/status.go

```diff
@@ -39,6 +39,14 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+const (
```

Comment:
I may have given conflicting comments now. Sorry if I did. But I think these could be merged with the tigerastatustypes.go.